twitchio/websocket.py

```diff
@@ -540,10 +570,13 @@ def _task_callback(self, data, task):
     async def part_action(self, channel: str, author: str, tags):
         log.debug('ACTION:: PART: %s', channel)
 
-        if author == self.nick:
+        if author.lower() == self.nick.lower():
```

Comment:
It would make more sense to just use .lower() in the constructor.

Reply:
That way self.nick is always in lower case form.

Reply:
Fixed in aeaf2cf